CONTRIBUTING.md

```diff
@@ -1,26 +1,42 @@
 
 # Reporting a bug
 
-Please check first that it [has not already been reported](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). Also consider [enabling debug mode](https://github.com/laurent22/joplin/blob/master/readme/debugging.md) before reporting the issue so that you can provide as much details as possible to help fix it.
+File bugs in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). Please follow these guidelines:
 
-If possible, **please provide a screenshot**. A screenshot showing the problem is often more useful than a paragraph describing it as it can make it immediately clear what the issue is.
+- Search existing issues first, make sure yours hasn't already been reported.
+- Don't use the issue tracker for support questions.
+- Consider [enabling debug mode](https://github.com/laurent22/joplin/blob/master/readme/debugging.md) before so that you can provide as much details as possible when reporting the issue.
+- Stay on topic, but describe the issue in detail so that others can reproduce it.
+- **Provide a screenshot** if possible. A screenshot showing the problem is often more useful than a paragraph describing it.
 
 # Feature requests
 
-Again, please check that it has not already been requested. If it has, simply **up-vote the issue** - the ones with the most up-votes are likely to be implemented. "+1" comments are not tracked.
+Please check that your request has not already been posted in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). If it has, **up-voting the issue** increases the chances it'll be noticed and implemented in the future. "+1" comments are not tracked.
 
-# Creating a pull request
+As a general rule, suggestions to _improve Joplin_ should be posted first in the [Joplin Forum](https://discourse.joplin.cozic.net/) for discussion.
 
-- If you want to add a new feature, consider asking about it before implementing it or checking existing discussions to make sure it is within the scope of the project. As a rule of thumb **if your change is likely to involve more than 50 lines of code, you should discuss it in the forum**, just so that you don't spend too much time implementing something that might not be accepted.
+Avoid listing multiple requests in one report in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). One issue per request makes it easier to track and discuss it.
 
-- Bug fixes are always welcome.
+# Contribute to the project
+
+## Contributing to Joplin's translation
+
+Joplin is available in multiple languages thanks to the help of its users. You can help translate Joplin to your language or keep it up to date. Please read the documentation about [Localisation](https://github.com/laurent22/joplin#localisation).
```

Comment:
Not sure, if we should use the repo URL or the web site: https://joplinapp.org/#localisation

@laurent22 what do you think?

Reply:
Ok, we can leave this for now. We can always change this later, if necessary.